gradlew.bat

```diff
@@ -8,14 +8,14 @@
 @rem Set local scope for the variables with windows NT shell

```

Comment:
You should never touch the gradlew-files (until new version comes) since they're generated automatically.